.travis.yml

```diff
@@ -1,8 +1,5 @@
 dist: trusty
-sudo: false
-services:
-  - mysql
-  - postgresql
+sudo: required
```

Comment:
So I needed to update the version of psycopg that was installed and I kept not being able to update it. I thought it was a system thing, so I switched to a newer dist. Then it turned out that the `sudo: false` build didn’t have mysql, so I switched to `sudo: required`. Then eventually I noticed that I was installing psycopg twice - I added an install of the newer version which was being overridden by the old install. Once I corrected that to just update the one install, it actually worked :-p